### PR TITLE
Add nuget package README

### PR DESCRIPTION
**Pull request by natke**
Not sure if the example output is too wordy?

### CODE REVIEW

File: nuget/PACKAGE.md
Comment (baijumeswani) [inline nuget/PACKAGE.md:1]: I don't think this is too wordy. But I was thinking we could have the same README.md for all packages and only change the language code snippet.

Anytime we update our README, we can update all language READMEs. 

We could do this by doing something like that. Showing for examples here, but can be replaced for readmes instead.

[<mark>Python Examples</mark>](https://github.com/microsoft/onnxruntime-genai/tree/main/examples/python) | [C# Examples](https://github.com/microsoft/onnxruntime-genai/tree/main/examples/csharp) | [C++ Examples](https://github.com/microsoft/onnxruntime-genai/tree/main/examples/c)

And the corresponding code snippet below. But this can be done separately later.

File: nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec [hunk 1]
```diff
@@ -19,7 +19,7 @@
   </metadata>
   <files>
     <file src="..\LICENSE" target="LICENSE" />
-    <file src="..\README.md" target="README.md" />
+    <file src="PACKAGE.md" target="PACKAGE.md" />
```
Comment (baijumeswani) [inline nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec:22]: Considering the location of the [python package description](https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/package_description.md), shall we add this package md file to https://github.com/microsoft/onnxruntime-genai/tree/main/src/csharp?

**Review by baijumeswani — APPROVED** [review]

File: nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec (hunk 1 above)
Comment (natke) [reply nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec:22]: Good idea. I can move it

File: nuget/PACKAGE.md
Comment (natke) [reply nuget/PACKAGE.md:1]: From the feedback from the .NET team, they like their READMEs to be .NET/C# specific. I know it is more for us but it is a better user experience

File: nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec (hunk 1 above)
Comment (natke) [reply nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec]: Moved

**Review by baijumeswani — APPROVED** [review]
